controller/contactUs.js

```diff
@@ -1,5 +1,27 @@
 const Contacts = require("../models/Contact")
 
+// Intern Validation rules
+const contactValidationRules = () => [
```

Comment:
you declared this and didn't use it in your create contact method

Reply:
removed